### ISSUE TITLE
Sign up: gender radio buttons not accessible

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#1849]
> Use fieldset or role="group" and aria-labelledby to label the group
> use label element to label each individual field (male, female, other)
